From khali@linux-fr.org Sun Feb  5 14:24:40 2006
Date: Sun, 5 Feb 2006 23:25:25 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 06/11] w83781d: Document the alarm and beep bits
Message-Id: <20060205232525.4a6377e1.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-w83781d-document-alarm-bits.patch

Document the individual alarm and beep bits of the w83781d driver.
Ideally we would offer a chip-independant interface for them, but
until it's done, it's only fair that we document the current
interface.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 Documentation/hwmon/w83781d |   19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

--- gregkh-2.6.orig/Documentation/hwmon/w83781d
+++ gregkh-2.6/Documentation/hwmon/w83781d
@@ -123,6 +123,25 @@ When an alarm goes off, you can be warne
 your computer speaker. It is possible to enable all beeping globally,
 or only the beeping for some alarms.
 
+Individual alarm and beep bits:
+
+0x000001: in0
+0x000002: in1
+0x000004: in2
+0x000008: in3
+0x000010: temp1
+0x000020: temp2 (+temp3 on W83781D)
+0x000040: fan1
+0x000080: fan2
+0x000100: in4
+0x000200: in5
+0x000400: in6
+0x000800: fan3
+0x001000: chassis
+0x002000: temp3 (W83782D and W83627HF only)
+0x010000: in7 (W83782D and W83627HF only)
+0x020000: in8 (W83782D and W83627HF only)
+
 If an alarm triggers, it will remain triggered until the hardware register
 is read at least once. This means that the cause for the alarm may
 already have disappeared! Note that in the current implementation, all
